src/snapred/backend/recipe/algorithm/ReductionAlgorithm.py

```diff
@@ -56,7 +56,7 @@ def PyExec(self):
         # 8 CreateGroupWorkspace      TODO: Assess performance, use alternative Andrei came up with that is faster
         groupingworkspace = self.mantidSnapper.CustomGroupWorkspace(
             "Creating Group Workspace...",
-            StateConfig=reductionIngredients.reductionState.stateConfig.json(),
+            FocusGroups=reductionIngredients.reductionState.stateConfig.json(),
```

Comment:
This is missing an accessor to focusgroups I think.

Reply:
You are correct